server/rest/src/main/java/org/infinispan/rest/resources/MetricsResource.java

```diff
@@ -71,19 +48,12 @@ public Invocations getInvocations() {
    }
 
    private CompletionStage<RestResponse> metrics(RestRequest restRequest) {
-      List<String> accept = responseMediaTypes(restRequest);
-
-      CompletableFuture<RestResponse> cf = CompletableFuture.supplyAsync(() -> {
+      return CompletableFuture.supplyAsync(() -> {
          RestResponseBuilder<NettyRestResponse.Builder> builder = new NettyRestResponse.Builder();
 
          try {
-            requestHandler.handleRequest(restRequest.path(), restRequest.method().name(),
-                                         accept.stream(), (status, message, headers) -> {
-                     builder.status(status).entity(message);
-                     for (String header : headers.keySet()) {
-                        builder.header(header, headers.get(header));
-                     }
-                  });
+            builder.header("Content-Type", TextFormat.CONTENT_TYPE_004);
+            builder.entity(metricsCollector.getBaseRegistry().scrape()+metricsCollector.getVendorRegistry().scrape());
```

Comment:
Nitpick: spacing doesn't match the codestyle. I would recommend importing the code style from https://github.com/infinispan/infinispan/tree/main/ide-settings

Reply:
Sure. I forgot to run the `Command+Option+L` here.